draft-jaju-httpbis-zstd-window-size.md

```diff
@@ -85,28 +90,10 @@ the "zstd" Content Encoding token.
 
 # Window Size
 
-Section 3.1.1.1.2 of {{RFC8878}} discusses window sizes in Zstandard.
-The window size provides guarantees about the minimum memory buffer required to
-decompress a frame. This information is important for decoders to allocate
-enough memory.
-
-The minimum window size is 1 KB. The maximum window size is (1<<41) +
-7*(1<<38) bytes, which is 3.75 TB.
-
-In general, larger window size values tend to improve the compression ratio, but
-at the cost of increased memory usage.
-
-To properly decode compressed data, a decoder will need to allocate a buffer of
-at least the window size bytes.
-
-In order to protect decoders from unreasonable memory requirements, a decoder is
-allowed to reject a compressed frame that requests a memory size beyond the
-decoder's authorized range.
```

Comment:
I think it helps to keep some variation of this text here for reading comprehension, especially since the intention of this document is to be the reference point for window size for zstd.